notebooks/official/sdk/SDK_Custom_Training_Python_Package_Managed_Text_Dataset_Tensorflow_Serving_Container.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>Heading needs to be in "sentence case". </p><p> </p><p>Custom training using Python package, managed text dataset, and TF Serving container</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>